governance/votes/Executive vote - January 28, 2022.md

```diff
@@ -0,0 +1,78 @@
+* **2,566,200 DAI** will be streamed to the [TechOps Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP29) at 0x2dC0420A736D1F40893B9481D8968E4D7424bC0B starting at 2022-02-01 and ending at 2023-01-31 as a result of their most recent [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP53).
+* **257,500 DAI** will be streamed to the[StarkNet Engineering Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP19) at 0x6D348f18c88D45243705D4fdEeB6538c6a9191F1 starting at 2022-02-01 and ending at 2022-07-31 as a result of their most recent [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP47).
+* **494,502 DAI** will be streamed to the [Strategic Finance Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP27) at 0xf737C76D2B358619f7ef696cf3F94548fEcec379 starting at 2022-02-01 and ending at 2022-07-31 as a result of their most recent [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP46).
+
+### Core Unit MKR Vesting Streams
+As per the [Strategic Finance Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP27) successful [MKR budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP48), a total of **480 MKR** will be vested to members of the [Strategic Finance Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP27) over the next 3 years if this executive proposal passes.
+* **240 MKR** will be streamed to Aes at 0xBC7fd5AA2016C3e2C8F0dBf4e919485C6BBb59e2 starting 2021-09-01 and ending at 2024-09-01 with a cliff date of 2022-09-01.
+* **240 MKR** will be streamed to SebVentures at 0xcc81578d163a04ea8d2eae6904d0c8e61a84e1bb starting 2021-04-01 and ending at 2024-04-01 with a cliff date of 2022-04-01.
+
+### Revoking Content Production DAI Budget
+As a result of this successful [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP49), the [Content Production Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP5) budget will be revoked if this executive proposal passes. Note that when a token stream is revoked, it will prevent new tokens accruing to the recipient, but will still allow the recipient access to tokens that have already accrued.
+
+### Replacing the Real World Finance DAI Budget Stream
+Given that the facilitator for the [Real World Finance Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP1) has changed, we recieved a request from the new facilitator to adjust the implementation of the existing Real World Finance DAI budget stream. This request is detailed here. Given that this is an administrative change and budget amounts remain the same, we elected to include it in this executive proposal. If this executive proposals passes the following will occur.
+* The existing Real World Finance DAI budget stream will be revoked.
+* A replacement stream will be setup, streaming **1,705,000 DAI** to 0x96d7b01Cc25B141520C717fa369844d34FF116ec starting at 2022-02-01 and ending at 2022-12-31.
```

Comment:
Maybe link to the budget MIP so we can see where the number came from?

Reply:
Think I'll leave this out. Important part is that it comes to the same amount, and Will's post should be illumination enough.